docs/src/content/docs/guides/split-payments.mdx

```diff
@@ -0,0 +1,331 @@
+---
+title: Split payments
+---
+
+import { CodeBlock, LinkOut } from '@interledger/docs-design-system'
+import { Tabs, TabItem } from '@astrojs/starlight/components'
+import HttpSig from '/src/partials/http-msg-sig-note.mdx'
+import ChunkedSnippet from '/src/components/ChunkedSnippet.astro'
+import Ts from '/src/partials/ts-prerequisites.mdx'
+
+# Build an e-Commerce platform with split-payments between merchants and platform
```

Comment:
I don't think we need this, especially as an H1 (the page title is automatically an H1). It essentially gives us an empty overview, since Starlight forces the first heading on the page to be "Overview".
<img width="600" alt="Screenshot 2025-03-05 at 8 23 53 AM" src="https://github.com/user-attachments/assets/7b3f6824-2fc6-46bd-ad4e-8ff8b6354fe7" />